code/ide.py

```diff
@@ -266,6 +275,9 @@ def ide_git_annotate():
     def ide_git_menu():
         """Show the git menu for IDE"""
 
+    def ide_toggle_explorer():
```

Comment:
Any reason not to just use ide_toggle_project?

Reply:
I felt there was enough Symantec difference to create this, but on checking a few IDE's I now have a different opinion.   
Yes in vscode this Explorer view maps to ide_toggle_project. However there is the ability to open an OS file explorer/manager and maybe this `ide_toggle_explorer`  should be that command `vscode> File:Reveal in file explorer` however it is not able to be toggled so it should finally be `ide_show_explorer`

additionally there are a number of other `ide_show_*` commands 
```
    def ide_show_explorer():
        """Show Explorer"""

    def ide_show_search():
        """Show Search"""

    def ide_show_source_control():
        """Show Source Control"""

    def ide_show_debug():
        """Show Debug"""

    def ide_show_extensions():
        """Show Extensions"""
```
I believe are hangovers from the merge of vscode.talon and ide.talon.   only the `ide_show_explorer` should stay for reason above.

I see you have done some great refactoring over weekend. I suggest I pull back this PR , cherry pick my updates on a new PR, I have also overweekend got my Linux Dev environment up and running ,so can test simultaneous against both platforms

Reply:
Yeah, I agree with the ide_show_* stuff, good catch.

i can see the value in that ide_toggle_explorer/ide_show_explorer action, but would suggest something like `ide_reveal_in_file_manager`. Arguably a bit more descriptive/generic. 

Pulling this PR back sounds reasonable to me. Sorry for the inconvenience due to the refactor, been sitting on it for too long.